app/src/main/java/org/simple/clinic/overdue/AppointmentSync.kt

```diff
@@ -14,9 +15,12 @@ import javax.inject.Named
 class AppointmentSync @Inject constructor(
     private val syncCoordinator: SyncCoordinator,
     private val repository: AppointmentRepository,
-    private val api: AppointmentSyncApiV2,
+    private val apiV2: AppointmentSyncApiV2,
+    private val apiV3: AppointmentSyncApiV3,
     private val userSession: UserSession,
-    @Named("last_appointment_pull_token") private val lastPullToken: Preference<Optional<String>>,
+    private val appointmentConfig: Single<AppointmentConfig>,
```

Comment:
We have started representing the config provider as an `Observable` instead of a `Single` in new places because we found that we are converting it to an `Observable` wherever we are using it anyway. Feel free to make this change if you like.

Reply:
Oh yes. I knew this. Sorry I forgot.

Reply:
Since `AppointmentConfig` was not introduced in this PR, I would like to make this change in a separate story.